src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2611,10 +2605,10 @@ namespace WaterToAirHeatPumpSimple {
         // calculate coil outlet state variables
         state.dataWaterToAirHeatPumpSimple->LoadSideOutletEnth =
             state.dataWaterToAirHeatPumpSimple->LoadSideInletEnth -
-            state.dataWaterToAirHeatPumpSimple->QLoadTotal / state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate;
+            state.dataWaterToAirHeatPumpSimple->QLoadTotal / (state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate / PartLoadRatio);
```

Comment:
This does look OK. QLoadTotal is the full load output. The outlet enthalpy would be Hin - (QLoadTotal/LoadSideMassFlowRate) * PLR so this equation also works.

Reply:
Is LoadSideOutletEnth the air side (line 2606)? And then below at line 2617 is OutletAirEnthalpy. What is the difference between these variables?

Reply:
Notice below at line 2617 that (PLR and 1-PLR) are used. This means that LoadSideOutletEnth should be the full load output (i.e., coil ON outlet air enthalpy). The coil is ON at PLR and OFF at 1-PLR so line 2617 averages the inlet and full load outlet enthalpy. I think this model is just reusing a variable used in WaterToAirHeatPump.cc that represents the load side refrigerant. So it appears from this that PLR is not needed at line 2608 for constant fan mode. This is confusing so be careful with these equations. At line 2608 the system could be either constant fan or cycling fan. Only divide by PLR for the cycling fan case. Maybe it would be better to move these equations below into the CyclingScheme block or don't even use LoadSideOutletEnth or LoadSideOutletDBTemp or LoadSideOutletHumRat and just do the calculations like this below where you know what the fan operating mode is.

Reply:
Historic coil model method (based on DX coil model):
- Constant fan case, air flow is full flow and outlet air conditions are average of coil ON state and coil OFF state.
- Cycling fan case, air flow is average flow (coil ON state and 0) and outlet air conditions are coil ON state.

Air flow is set by the parent and should be correct coming in to this model. The model then needs to account for operating conditions (constant vs cycling fan) to model the coil at full output.

Reply:
So at line 2617 below (cont fan) the model is reporting the average outlet condition, at line 2629 (cyc fan) the model is reporting full load outlet condition.

Reply:
Thanks, @rraustad . It looks to me that the current changes agreed with what you said for constant and cycling fan behaviors. 
Line 2608 is calculating the full load outlet air enthalpy and for these two types of fan operations: 
  1. line 2617-2626 are averaging for constant fan and reporting the full air flow rate, 
  2. line 2629-2632 are directly using outlet conditions but applying air flow rate with PLR. 

Did I miss anything? Is there anything that I changed that might break things unexpectedly? 

And for the variables,  there're more than one place using multiple variables that I don't understand what's the difference. `OutletAirEnthalpy` and `LoadSideOutletEnth` is one example,  `LoadSideMassFlowRate` and `AirMassFlowRate` is another. It's confusing when I want to track var values. Maybe those were useful at some point but not now, I plan to clean them up after the fix.

Reply:
QLoadTotal is the full load output, calculated at line 2554 as: 
state.dataWaterToAirHeatPumpSimple->QLoadTotal = TotalCapRated * CapacityPerformanceCurveOutput

If you divide by PartLoadRatio at line 2608 when constant fan mode is used you will get less than the full load outlet enthalpy difference. 

If you did this you would be OK:

    Real64 PLR = 1.0;
    if (CyclingScheme == CycFanCycCoil) PLR = PartLoadRatio;

            state.dataWaterToAirHeatPumpSimple->LoadSideOutletEnth =
                state.dataWaterToAirHeatPumpSimple->LoadSideInletEnth -
            state.dataWaterToAirHeatPumpSimple->QLoadTotal / (state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate / PLR);